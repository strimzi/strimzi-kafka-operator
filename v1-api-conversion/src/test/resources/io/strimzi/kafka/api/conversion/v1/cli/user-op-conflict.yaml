apiVersion: kafka.strimzi.io/v1beta2
kind: KafkaUser
metadata:
  name: my-user
  labels:
    strimzi.io/cluster: my-cluster
spec:
  authentication:
    type: tls
  authorization:
    type: simple
    acls:
      - resource:
          type: topic
          name: my-topic
          patternType: literal
        operation: Write
        operations:
          - Read
          - Describe