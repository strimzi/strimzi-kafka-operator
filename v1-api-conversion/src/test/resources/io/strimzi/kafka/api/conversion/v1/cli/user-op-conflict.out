---
apiVersion: kafka.strimzi.io/v1
kind: KafkaUser
metadata:
  name: my-user
  labels:
    strimzi.io/cluster: my-cluster
spec:
  authentication:
    type: tls
  authorization:
    type: simple
    acls:
    - resource:
        type: topic
        name: my-topic
      operations:
      - Read
      - Describe